src/event-listeners/index.js

```diff
@@ -37,10 +37,15 @@ import {
   changeTranslate,
   changeScale
 } from '../viewport/actions'
+import drawSelectBox from '../drawing/drawSelectBox'
```

Comment:
@hector0513 right direction, but import this into `src/drawing/index.js` instead, and start the drawing process from within the `render` function of that file, along with the rest of the calls to drawing